.github/workflows/release.yml

```diff
@@ -115,3 +115,14 @@ jobs:
           DISCORD_WEBHOOK_TOKEN: ${{ secrets.DISCORD_WEBHOOK_TOKEN }}
           AUR_KEY: ${{ secrets.AUR_KEY }}
         run: task goreleaser
+      - name: get version
+        if: success() && startsWith(github.ref, 'refs/tags/v')
+        run: echo "RELEASE_TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+      - name: notify goreleaser-cross with new release
+        if: success() && startsWith(github.ref, 'refs/tags/v')
+        uses: peter-evans/repository-dispatch@v2
```

Comment:
maybe put in another job, like `trigger-generate` on line 16?

also, we could use the same action for both triggers, the other one is using `benc-uk/workflow-dispatch`... I don't have strong feelings about any of them, just think it reduces our burden to depend on only one of them

Reply:
makes sense, need to test how `benc-uk/workflow-dispatch` work with `gorealeser-cross` tho

Reply:
yeah, it works, i forced pushed changes below